frontend/src/store.js

```diff
@@ -0,0 +1,35 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import { routerMiddleware } from 'react-router-redux';
+import thunk from 'redux-thunk';
+import createHistory from 'history/createBrowserHistory';
+import rootReducer from './modules';
+
+export const history = createHistory();
+
+const initialState = {};
+const enhancers = [];
+const middleware = [
+  thunk,
+  routerMiddleware(history)
+];
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
no, you don't have to do anything...

This gets set by the webpack development server.  Right now this is just scaffolding, but when I start pushing data through the app, I can show you how this flows through the component and how to use Chrome dev tools to inspect it.